M2/Macaulay2/d/actors4.d

```diff
@@ -1313,24 +1313,24 @@ toRRi(e:Expr):Expr := (
                	       when s.1 is y:ZZcell do toExpr(toRRi(x.v,y.v))
                                 is y:QQcell do toExpr(toRRi(x.v,y.v))
                                 is y:RRcell do toExpr(toRRi(x.v,y.v))
-                    	        else WrongArg(1,"a pair of integral, rational, or real numbers"))
+                    	        else WrongArg(1,"a pair of integral, rational, or real numbers, with a precision"))
```

Comment:
Adding "with a precision" makes it sound like the user should provide a real number and a precision, two things.  I think it was fine the way it was.